.travis.yml

```diff
@@ -140,18 +140,42 @@ jobs:
     dist: trusty
     env: BUILD_TYPE=Release COMPILER=gcc-9
     name: gcc-9 release
-  - &ClangDebug
+  - &Clang4Debug
     os: linux
-    compiler: clang
+    compiler: clang-4.0
     dist: trusty
-    env: BUILD_TYPE=Debug
-    name: clang debug
-  - &ClangRelease
+    env: BUILD_TYPE=Debug COMPILER=clang-4.0
+    name: clang-4 debug
+  - &Clang4Release
     os: linux
-    compiler: clang
+    compiler: clang-4.0
+    dist: trusty
+    env: BUILD_TYPE=Release COMPILER=clang-4.0
+    name: clang-4 release
+  - &Clang5Debug
+    os: linux
+    compiler: clang-5.0
+    dist: trusty
+    env: BUILD_TYPE=Debug COMPILER=clang-5.0
+    name: clang-5 debug
+  - &Clang5Release
+    os: linux
+    compiler: clang-5.0
+    dist: trusty
+    env: BUILD_TYPE=Release COMPILER=clang-5.0
+    name: clang-5 release
+  - &Clang8Debug
+    os: linux
+    compiler: clang-8
+    dist: trusty
+    env: BUILD_TYPE=Debug COMPILER=clang-8
+    name: clang-8 debug
+  - &Clang8Release
```

Comment:
For my understanding: why clang versions 4,5,8 but not 6,7 ?

Reply:
To give a compromise between the number of targets and how long it takes to build. There's no technical reason, but clang4 is the oldest we can support, clang5 was used for IWYU, and clang8 since something broke in going from 7 to 8. Does that make sense?

Reply:
Yep! I assumed it was something like this. Thanks for clarifying.

Reply:
Ah, good point! Will do :)